pre_push.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+"""Run static analysis on the project."""
```

Comment:
Nice job on making this OS-compatible

Reply:
I'm not going to take credit for it as it was copied from PRAW.